administrator/components/com_media/views/file/tmpl/default.php

```diff
@@ -13,8 +13,8 @@
 JHtml::_('jquery.framework');
```

Comment:
It's still needed because the edit.js file is jQuery dependent. If somebody with more knowledge can fix that, would be nice to remove it.